packages/playwright/src/worker/testTracing.ts

```diff
@@ -302,6 +302,7 @@ function generatePreview(value: any, visited = new Set<any>()): string {
 }
 
 async function mergeTraceFiles(fileName: string, temporaryTraceFiles: string[]) {
+  temporaryTraceFiles = temporaryTraceFiles.filter(file => fs.existsSync(file));
```

Comment:
nit: everything else in this method uses async, so perhaps `fs.promises.access()`?

Reply:
Discussed offline: let's stay sync for simplicity.